backend/benefit/companies/tests/test_api.py

```diff
@@ -129,6 +132,8 @@ def test_get_company_from_yrtti(
     )
     company_data = CompanySerializer(company).data
     assert response.data == company_data
+    assert response.data["company_form_code"] == 29
```

Comment:
`assert response.data["company_form_code"] == ASSOCIATION_FORM_CODE` would be more readable.